### PR TITLE
feat: cookies

### PR DESCRIPTION
**Pull request by Prerna290**
## Type
- ( ) Fix
- ( ) Refactor
- ( ) Style
- ( ) Docs
- (X) Feat
- ( ) Hotfix

## Screenshots
image
image

## Bundle Size
4.88 MB

### CODE REVIEW

File: apps/shared-components/cookie-consent/cookie-consent.component.ts [hunk 1]
```diff
@@ -12,13 +13,26 @@ import { SeoService } from 'apps/shared-services/seo.service';
 export class CookieConsentComponent implements OnInit {
   cookieConstent = false;
   isBrowser;
+  showPreferncesButton = false;
+  allowNecessary = true;
+  allowAnalytics = true;
+  allowMarketing = true;
```
Comment (ArshdeepGrover) [inline apps/shared-components/cookie-consent/cookie-consent.component.ts]: is this needed?

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/shared-components/cookie-consent/cookie-consent.component.ts (hunk 1 above)
Comment (Prerna290) [reply apps/shared-components/cookie-consent/cookie-consent.component.ts]: removed

File: apps/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/cookie-preferences/cookie-preferences.component.scss [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.cookie-preference-container {
+  height: 620px;
```
Comment (ArshdeepGrover) [inline apps/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/cookie-preferences/cookie-preferences.component.scss]: try to use tailwindCss

**Review by ArshdeepGrover — CHANGES_REQUESTED** [review]

File: apps/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/cookie-preferences/cookie-preferences.component.scss (hunk 2 above)
Comment (Prerna290) [reply apps/commudle-admin/src/app/feature-modules/users/components/public-profile/user-basic-details/cookie-preferences/cookie-preferences.component.scss]: Done

**Review by ArshdeepGrover — APPROVED** [review]

**Pull request merged**